system/views/formcontrols/spinner/index.cfm

```diff
@@ -24,5 +24,5 @@
 </cfscript>
 
 <cfoutput>
-	<input type="number" id="#inputId#" class="#inputClass#" name="#inputName#" value="#value#"<cfif isNumeric( minValue )> min="#minValue#"</cfif><cfif isNumeric( maxValue )> max="#maxValue#"</cfif> step="#step#" tabindex="#getNextTabIndex()#" #htmlAttributes# />
+	<input type="number" id="#inputId#" class="#inputClass#" name="#inputName#" value="#value#"<cfif isNumeric( minValue )> min="#minValue#"</cfif><cfif isNumeric( maxValue )> max="#maxValue#"</cfif><cfif Len( Trim( placeholder ) )> placeholder="#placeholder#"</cfif> step="#step#" tabindex="#getNextTabIndex()#" #htmlAttributes# />
```

Comment:
This placeholder should be encoded for output given its dynamic nature (could be translated into something that breaks html). i.e.

```cfc
#encodeForHtmlAttribute( placeholder )#
```